libraries/AP_DroneCAN/AP_DroneCAN.cpp

```diff
@@ -138,6 +138,13 @@ const AP_Param::GroupInfo AP_DroneCAN::var_info[] = {
     // @Range: 1024 16384
     // @User: Advanced
     AP_GROUPINFO("POOL", 8, AP_DroneCAN, _pool_size, DRONECAN_NODE_POOL_SIZE),
+
+    // @Param: ESC_RM
+    // @DisplayName: Bitmask for output channels for reversible ESCs over DroneCAN.
+    // @Description: Bitmask with one set for each output channel that uses a reversible ESC over DroneCAN. Reversible ESCs use both positive and negative values in RawCommands, with positive commanding the forward direction and negative commanding the reverse direction.
+    // @Bitmask: 0: ESC 1, 1: ESC 2, 2: ESC 3, 3: ESC 4, 4: ESC 5, 5: ESC 6, 6: ESC 7, 7: ESC 8, 8: ESC 9, 9: ESC 10, 10: ESC 11, 11: ESC 12, 12: ESC 13, 13: ESC 14, 14: ESC 15, 15: ESC 16, 16: ESC 17, 17: ESC 18, 18: ESC 19, 19: ESC 20, 20: ESC 21, 21: ESC 22, 22: ESC 23, 23: ESC 24, 24: ESC 25, 25: ESC 26, 26: ESC 27, 27: ESC 28, 28: ESC 29, 29: ESC 30, 30: ESC 31, 31: ESC 32
+    // @User: Advanced
+    AP_GROUPINFO("ESC_RM", 9, AP_DroneCAN, _esc_rm, 0),
```

Comment:
Sadly, only the default values will work for all 32 bits so we need to make sure the default is a define instead of 0. Mavlink sends params as floats so we only get 24bit integer bits reliably.

Reply:
We call this RVMASK in blheli, I think you should use RV rather than RM

Reply:
Yeah, RV probably makes more sense, I'll change this to ESC_RV to be more consistent. Looking at the BLHeli parameters, I see that the RVMASK parameter also uses 0 directly as its default, so I'm still a little bit confused about how to properly address the concerns from the previous comment.
![image](https://github.com/ArduPilot/ardupilot/assets/95372252/9f064849-0d3d-497a-80b8-4f6b608b8efa)